lib/diagnostic.js

```diff
@@ -29,8 +33,14 @@ const sum = function (/* previous, current */) {
 // in the array returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.some`
-const any = function (/* array, callback */) {
+const any = function (array, callback) {
+  for (let i = 0; i < array.length; i++) {
+    if (callback(array[i])) {
```

Comment:
As specified in the instructions, we want ```callback``` to take ```element```, ```index```, and ```array``` as arguments.  This condition should be ```callback(array[i], i, array)```.  While it may seem extraneous, the purpose is to have everyone programming to a particular interface, since others may use your code or have particular specifications that they want you to implement.